client/grpc/grpc_test.go

```diff
@@ -13,13 +13,12 @@ import (
 	"testing"
 	"time"
 
+	"github.com/s7techlab/hlf-sdk-go/api/config"
```

Comment:
please use 3 groups imports (goimports -local)

Reply:
fixed in last commit